src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -3445,102 +3476,108 @@ private bool TryReadInternal(bool setTimeout, out bool more)
                         _sharedState._columnDataBytesRemaining = -1; // unknown
                         _lastColumnWithDataChunkRead = -1;
 
-                        if (!_haltRead)
+                    if (!_haltRead)
```

Comment:
Same as above

Reply:
Reformatted the method to cleanup the discrepancies caused by #ifdefs